python/vowpalwabbit/pyvw.py

```diff
@@ -1898,35 +1745,23 @@ def process(self, df):
 
 class Feature(object):
     """The feature type for the constructor of DFtoVW"""
-
-    name = AttributeDescriptor("name", expected_type=(str, int, float))
     value = AttributeDescriptor("value", expected_type=(str, int, float))
 
-    def __init__(
-        self, value, name=None, value_from_df=True, name_from_df=False
-    ):
+    def __init__(self, value, name=None):
         """
         Initialize a Feature instance.
 
         Parameters
         ----------
-        value : str/float/int
-            The value of the feature.
-        name : str/float/int, optional
-            The name of the feature.
-        value_from_df: bool (default: True)
-            True if the value refers to a column in the dataframe and False if
-            the value is a constant.
-        name_from_df: bool (default: False)
-            True if the name refers to a column in the dataframe and False if
-            the name is a constant.
+        value : str
+            The column name with the value of the feature.
+        name : str
+            The column name with the name of the feature.
```

Comment:
The explanation is a bit misleading here, it's actually not a column name. It is just a simple string that is used to name the feature (with a different name than its colname). Maybe this arg should be called rename instead.

Reply:
what about something like name_override? and with some explanation like "set a string name to use instead of using colname of value" (feel free to paraphrase or improve). Also, I'm assuming that value is required always to build a feature and name is optional right? maybe adding that to the description is helpful. Thoughts?

Reply:
Ok I change it to `rename_feature` which is quite straightforward IMO, I change the docstring accordingly